incubator/java-microprofile/README.md

```diff
@@ -1,47 +1,63 @@
-# Java Microprofile
+# Java Microprofile Stack
 
-A stack is the foundation of your application which provides a prototype platform definition for Java Microprifile based projects.
+The Java Microprofile stack provides a consistent way of developing microservices based upon the [Eclipse MicroProfile® specifications](https://microprofile.io). This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, which includes OpenJDK with container-optimizations in OpenJ9.
 
-The Java Microprofile stack uses a parent POM to manage dependency versions and provide required capabilities and plugins whilst using Maven. Specifically, this stack enables adoptopenjdk/openjdk8-openj9 and Open Liberty version `19.0.0.5`.
+The Java Microprofile stack uses a parent Maven project object model (POM) to manage dependency versions and provide required capabilities and plugins.
+
+This stack is based on `adoptopenjdk/openjdk8-openj9` and `Open Liberty v19.0.0.5`. It provides live reloading during development by utilizing `loose application` capabilities.
 
 ## Templates
 
-Templates are used to create your local project and start your developement. When initializing your project you will be provided with a open liberty sample application by default.
+Templates are used to create your local project and start your development. When initializing your project you will be provided with an Open Liberty template application.
+
+The default template provides a `pom.xml` file that references the parent POM defined by the stack and enables Liberty features that support [Eclipse MicroProfile 2.2](https://openliberty.io/docs/ref/feature/#microProfile-2.2.html). Specifically, this template includes:
 
-The following capabilities are available from the Java Microprofile templates:
+### MicroProfile Health
```

Comment:
Do we need the word Microprofile in front of the titles Health, Metrics and OpenAPI, as I understand e.g. what OpenAPI is but don't get what we mean by "Microprofile OpenAPI"

Reply:
MicroProfile have specific features like `MicroProfile Health` or `mpHealth` that provide capabilities for health checking.

Reply:
removed to keep it less wordy and simple